Makefile

```diff
@@ -8,6 +8,11 @@
 
 .DEFAULT_GOAL := nitro-cli
 
+ARCH_x86_64      = x86_64
```

Comment:
pull requests checks are failing, could you have a look to see what is going on ?

Reply:
Sorry, forgot to mention yesterday. The bump to Rust 1.49.0 bump shall cause `make nitro-clippy` and `make nitro-format` to fail. We shall fix the 1.49.0 issues in a separate PR and this PR should pass after I'll rebase and repush.